superduperdb/db/base/artifact.py

```diff
@@ -105,10 +108,13 @@ def rewrite(t):
             info, serializer = t.get('info'), t['serializer']
             artifact = make(file_id=file_id, serializer=serializer)
```

Comment:
in case of lazy=True, 

make will return a None object, hence artifact = None here,


can we create a EmptyArtifact Class which this `make` can return.

Reply:
I'm not sure I understand.